src/lib/data/stores/localization.js

```diff
@@ -1,10 +1,12 @@
 import { derived } from 'svelte/store';
 import { userSettings } from './setting';
+import { getLanguages } from '$lib/data/language';
 import config from '../config';
 
 /** localization */
-export const languageDefault = config.translationMappings.defaultLang;
-export const languages = Object.keys(config.interfaceLanguages.writingSystems);
+export const languageDefault =
```

Comment:
languageDefault is just an alias for `config.translationMappings.defaultLang`.  

If you look at the current usage in SideBar.svelte, it is used to find the fallback translation in case the MenuItem.title doesn't have a translation based on the current language.